api/src/wfl/executor.clj

```diff
@@ -340,7 +340,8 @@
   [{:keys [details] :as _executor}]
   (let [query "SELECT COUNT(*) FROM %s
                WHERE consumed IS NULL
-               AND   status   NOT IN ('Failed', 'Aborted')"]
+               AND  (status   IS NULL OR
+                     status   NOT IN ('Failed', 'Aborted'))"]
```

Comment:
These don't need to be in parenthesis.

Reply:
I think you're right given our expectations of the problem domain right now (ex. if a row has been consumed then that implies that its status is 'Succeeded' and not null).  But I'll keep them since boolean logic will evaluate the `AND` first, and the parens will guard against weird behavior if this clause is altered further in the future.